builds/misc/addons-publish.yaml

```diff
@@ -20,13 +20,19 @@ jobs:
           - checkout: self
             clean: true
             fetchDepth: 100
-        
-          - task: Bash@3
-            displayName: Log into Registries
+
+          # Both docker logins needed for if we need to test this job. In this case images should go to edgebuilds.
+          - task: Docker@2
+            displayName: Docker login
```

Comment:
Could you update the display name to be more indicative of what CR is being log into please?